src/user_templates_api/templates/jupyter_lab/templates/squidpy/metadata.json

```diff
@@ -8,5 +8,13 @@
     ], 
     "is_multi_dataset_template": true,
     "template_format": "jinja",
+    "examples": [
+        {
+            "example_id": 0,
```

Comment:
Would it make sense to add a `name` or `title`?

Reply:
Hm perhaps, though I wonder what would be a sensible title for a combination of datasets. If these have some common factors (e.g. the first example is of type A / organ A / etc, and the second of type B / organ B / etc), this makes sense, but I think most of these examples would just be 'any collection', so what would be a good title for that?

Reply:
I'll ask Tiffany to take a look. I think we would want some sort of title or name for UI purposes, but maybe there is a way to present these examples without one.

Reply:
Given that there could be multiple examples for a template, I would prefer a title to differentiate the examples from each other. Just something that describes at a quick glance what this example is achieving

Reply:
I presure that most will just have one set as an example. Is it possible that these don't have a title? I think there will otherwise just be a lof of duplication between description template, description example, and title example. (unless this is not an issue?)

Reply:
If the title is going to be almost a duplication of what we already have in the template name/description and example description, I would be fine not having a title

Reply:
@thomcsmits Sorry for the back-and-forth, working through the mockup, I would prefer a title for these examples. Just think of it as "what would you name this workspace", so for example for this one something like "Slide-seq dataset analysis" or something similar.